extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -21,8 +21,8 @@ window.MozTranslationNotification = class extends MozElements.Notification {
             </menulist>
             <label value="&translation.translateThisPage.label;"/>
             <button class="notification-button primary" label="&translation.translate.button;" anonid="translate" oncommand="this.closest('notification').translate();"/>
```

Comment:
I will let @andrenatal comment on this as he knows about it.

Reply:
No, we didn't since it's already working and that's how the original file is implemented: https://github.com/mozilla/gecko-dev/blob/d36cf98aa85f24ceefd07521b3d16b9edd2abcb7/browser/components/translation/content/translation-notification.js#L31.

Reply:
I think this is out of the scope of this PR and maybe should be filed at Gecko?

Reply:
It's already on file https://bugzilla.mozilla.org/show_bug.cgi?id=1734220

Please note that you won't be able to make any change there, unless it's migrated to Fluent. That's why I asked if someone looked into it, now that I see this code fragment used in the add-on.